aiodogstatsd/client.py

```diff
@@ -257,6 +257,7 @@ def timeit(
         *,
         tags: Optional[typedefs.MTags] = None,
         sample_rate: Optional[typedefs.MSampleRate] = None,
+        threshold_ms: typedefs.MValue = 0,
```

Comment:
To my mind, it's not a backward compatible change, I prefer to set default value as `None` and then:
```python
if threshold_ms is not None and value > threshold_ms:
    self.timing(...)
```